### ISSUE TITLE
Review to see if this is still happening: [Other - Accessibility] Odd NVDA handling of errors, may be USWDS component bug. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#63095]
#### VFS team name
Veteran Facing Forms

#### VFS product name
21P-0847

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
On required text inputs and memorable dates (but not radio groups), there's an odd behavior with NVDA where, if you attempt to continue with an invalid or missing response, the input label is read three times before reading the error message.

The impact here is minimal, but it could be annoying to a screen reader user.

### Link, screenshot or steps to recreate
Video from testing with NVDA on Windows. The issue doesn't appear to happen with JAWS. Have not tested with VoiceOver or any mobile screen readers.

https://github.com/department-of-veterans-affairs/va.gov-team/assets/6844775/9e228b2b-b9b2-435d-a65a-7a77812ce159

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
I haven't been able to determine a cause, but I've got to think this is somehow a USWDS component bug. I checked an older form elsewhere on VA.gov and couldn't reproduce that behavior.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jeana-adhoc** [comment]
There seems to be a number of bug tickets filed with NVDA regarding the [double/triple speaking issue](https://github.com/nvaccess/nvda/issues?q=label%3Abug%2Fdouble-speaking+) I'm going to go through them and verify that our issue is similar and see if there are any suggestions on what to do about it. I also want to test the USWDS design system and see if things are being repeated there as well.

**Comment by jeana-adhoc** [comment]
I've confirmed that this seems to be only happening in our implementation of the component.

Both [USWDS](https://designsystem.digital.gov/components/text-input/) and [VADS Storybook](https://design.va.gov/storybook/?path=/docs/uswds-va-text-input--error) examples do *not* have this repeating text in NVDA.

**Comment by jeana-adhoc** [comment]
In looking at this some more, I see that our mock-form that has a [full name pattern where the label+ warning message *is not* repeated multiple times](https://staging.va.gov/mock-simple-forms-patterns/text-input-full-name). The code appears to be the same across the two forms. 

I'll ask the engineers on our team if they can spot any differences.
